recipe/meta.yaml

```diff
@@ -1,24 +1,23 @@
-{% set clang_variant = os.environ.get('CLANG_VARIANT', 'flang') %}
-{% set flang_commit = "1192ff0775ddf94f7944dc40181406c088b4d575" %}
-{% set llvm_version = "11.0.1" %}
-{% set version = llvm_version %}
-{% set sha256 = "e4fa46a0b29cabf4c4583231b79f923d6296efc4196099cbaebf5828f03a2607" %}
-{% set build_number = "20210131" %}
+{% set version = "17.0.0.rc1" %}
 
 package:
   name: flang-split
   version: {{ version }}
 
 source:
-  url: https://github.com/xoviat/flang/archive/{{ flang_commit }}.tar.gz
-  sha256: {{ sha256 }}
+  url: https://github.com/llvm/llvm-project/releases/download/llvmorg-{{ version.replace(".rc", "-rc") }}/llvm-project-{{ version.replace(".rc", "rc") }}.src.tar.xz
+  sha256: 01b80625c131b8281dcf76a4e6395d10f45835c60e27357c92abf5a98c9a0337
+  patches:
+    # link compiler-rt libs (upstream's find_compiler_rt_library doesn't work correctly);
+    # probably related to https://github.com/llvm/llvm-project/issues/63286
+    - patches/0001-always-link-in-compiler-rt-builtins-on-windows-apple.patch
```

Comment:
Is this still needed?

Reply:
IIRC, that was necessary to build at all, not just to run. I can try commenting it out, but I'll do that after.

Reply:
Well, perhaps that was only the case as of 16.0 (when I added that patch; I just rebased for 17), seems the build still passes without it 🥳